rasa/core/policies/ensemble.py

```diff
@@ -91,590 +220,318 @@ def _check_for_important_policies(self) -> None:
+    coincide (i.e. if we cannot distinguish two predictions using 1.-4., then
+    the first prediction is considered to be "better").
 
-    @staticmethod
-    def _get_updated_epochs(
-        policy_cls: Type[Policy],
-        config_for_policy: Dict[Text, Any],
-        finetuning_epoch_fraction: float,
-    ) -> Optional[int]:
-        if EPOCHS in config_for_policy:
-            epochs = config_for_policy[EPOCHS]
-        else:
-            try:
-                epochs = policy_cls.defaults[EPOCHS]
-            except (KeyError, AttributeError):
-                return None
-        return math.ceil(epochs * finetuning_epoch_fraction)
+    The list of events in the final prediction will contain all mandatory
+    events contained in the given predictions, the optional events given in the
+    "best" prediction, and `DefinePrevUserUtteredFeaturization` event (if the
+    prediction was made for a sequence of events ending with a user utterance).
+    """
 
     @classmethod
-    def load(
+    def create(
         cls,
-        path: Union[Text, Path],
-        new_config: Optional[Dict] = None,
-        finetuning_epoch_fraction: float = 1.0,
-    ) -> "PolicyEnsemble":
-        """Loads policy and domain specification from disk."""
-        metadata = cls.load_metadata(path)
-        cls.ensure_model_compatibility(metadata)
-        policies = []
-        for i, policy_name in enumerate(metadata["policy_names"]):
-            policy_cls = registry.policy_from_module_path(policy_name)
-            dir_name = f"policy_{i}_{policy_cls.__name__}"
-            policy_path = os.path.join(path, dir_name)
-
-            context = {}
-            if new_config:
-                context["should_finetune"] = True
-
-                config_for_policy = new_config["policies"][i]
-                epochs = cls._get_updated_epochs(
-                    policy_cls, config_for_policy, finetuning_epoch_fraction
-                )
-                if epochs:
-                    context["epoch_override"] = epochs
-
-            policy = policy_cls.load(policy_path, **context)
-            cls._ensure_loaded_policy(policy, policy_cls, policy_name)
-            if policy is not None:
-                policies.append(policy)
-        ensemble_cls = rasa.shared.utils.common.class_from_module_path(
-            metadata["ensemble_name"]
-        )
-        fingerprints = metadata.get("action_fingerprints", {})
-        ensemble = ensemble_cls(policies, fingerprints)
-        return ensemble
+        config: Dict[Text, Any],
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+    ) -> GraphComponent:
+        """Creates a new instance (see parent class for full docstring)."""
+        return cls()
 
-    @classmethod
-    def from_dict(cls, policy_configuration: Dict[Text, Any]) -> List[Policy]:
-        import copy
+    def __str__(self) -> Text:
+        return f"{self.__class__.__name__}()"
```

Comment:
This is so generic, shouldn't it be part of the `PolicyPredictionEnsemble` base class?

Reply:
There's not really a point in moving that there because the child class won't use str(super()) and you'll never instantiate the (abstract) base class (I should give that one an "ABC" though :D)

Reply:
(and the reason I'm talking about str(super()) is that other ensembles might have configs that we'd like to printout)